packages/driver/cypress/integration/cypress/currentTest_spec.js

```diff
@@ -0,0 +1,44 @@
+const rootResult = Cypress.currentTest
+
+describe('currentTest', () => {
+  const suiteResult = Cypress.currentTest
+
+  before(() => {
+    expectMatchingCurrentTitleInHook()
+  })
+
+  beforeEach(() => {
+    expectMatchingCurrentTitleInHook()
+  })
+
+  after(() => {
+    expectMatchingCurrentTitleInHook()
+  })
+
+  afterEach(() => {
+    expectMatchingCurrentTitleInHook()
+  })
+
+  it('returns current test runnable properties', () => {
+    expect(Cypress.currentTest.title)
+    .is.a('string')
+    .eq(cy.state('runnable').title)
```

Comment:
`title` could be the same wrong string for both, so this test could use an assertion against a string literal to be more regression-proof

```suggestion
    .eq(cy.state('runnable').title)
    .eq('returns current test runnable properties')

```